docs/pages/docs/themes.mdx

```diff
@@ -75,30 +75,36 @@ A more detailed explanation of each class can be found in the [Class Diagram](#c
 
 ## Customize Themes
 
-It's recommended to use the predefined themes as a starting point and customize them with the `copyWith(...)` method to suit your needs.
+It's recommended to use the predefined themes as a starting point and customize them with the `inhert(...)`[] constructor
+and `copyWith(...)` method to suit your needs.
 
-```dart filename="main.dart" {3-4, 8, 11, 14}
+```dart filename="main.dart" {3-14, 17-23}
 @override
-Widget build(BuildContext context) => FTheme(
-      data: FThemes.zinc.light.copyWith(
-        colorScheme: FThemes.zinc.light.colorScheme.copyWith(
-          primary: const Color(0xFF0D47A1), // dark blue
-          primaryForeground: const Color(0xFFFFFFFF), // white
-        ),
-        typography: FThemes.zinc.light.typography.copyWith(
-          defaultFontFamily: 'Roboto',
-        ).scale(sizeScalar: 0.8),
-        style: FThemes.zinc.light.style.copyWith(
-          borderRadius: BorderRadius.zero,
-        ),
-        cardStyle: FThemes.zinc.light.cardStyle.copyWith(
-          decoration: FThemes.zinc.light.cardStyle.decoration.copyWith(
-            borderRadius: const BorderRadius.all(Radius.circular(8)),
-          ),
+Widget build(BuildContext context) {
+  final theme = FThemeData.inherit(
+    colorScheme: FThemes.zinc.light.colorScheme.copyWith(
+      primary: const Color(0xFF0D47A1), // dark blue
+      primaryForeground: const Color(0xFFFFFFFF), // white
+    ),
+    typography: FThemes.zinc.light.typography.copyWith(
+      defaultFontFamily: 'Roboto',
+    ).scale(sizeScalar: 0.8),
+    style: FThemes.zinc.light.style.copyWith(
+      borderRadius: BorderRadius.zero,
+    ),
+  );
+
+  return FTheme(
+    data: theme.copyWith(
+      cardStyle: theme.cardStyle.copyWith(
```

Comment:
Might be a good idea to add a call out above, that the copyWith here is with the `theme` and not `FThemes.zinc`. A quick explanation help!

Reply:
Good catch!